presto-main/src/main/java/com/facebook/presto/execution/DataDefinitionExecution.java

```diff
@@ -237,7 +238,8 @@ public String explain(Statement statement, List<Expression> parameters)
                 String query,
                 Session session,
                 Statement statement,
-                List<Expression> parameters)
+                List<Expression> parameters,
+                GlobalProperties globalProperties)
```

Comment:
This argument seems to be unused? Do you need to pass it to DataDefinitionExecution?